docs/.vitepress/sidebars.mjs

```diff
@@ -323,33 +360,9 @@ export const guidesSidebar = [
     text: `<span style="display: flex; flex-direction: row; align-items: center; gap: 7px;">Share ${cube01Icon()}</span>`,
     items: [
       {
-        text: "Previews",
+        text: `<span style="display: flex; flex-direction: row; align-items: center; gap: 7px;">Previews ${xcodeProjCompatibleBadge()}</span>`,
```

Comment:
Should we also add a note in the `Previews` page as well, as we do for `Coming soon`?

![image](https://github.com/user-attachments/assets/2fbd0c1b-9050-463a-b14b-dae6a86a0ec5)
![image](https://github.com/user-attachments/assets/a96114c6-8d43-4d1e-b550-59b0655ecf5b)

Reply:
Yes! I added it there.